script/run-cypress-tests-circle.sh

```diff
@@ -11,5 +11,5 @@ else
   reporterArgs="--reporter cypress-multi-reporters --reporter-options \"configFile=config/cypress-reporters.json\""
 
   # Start the web server & run Cypress tests.
-  yarn start-server-and-test "node src/platform/testing/e2e/test-server.js --buildtype=vagovprod --port=3001 > /dev/null 2>&1" :3001 "yarn cy:run $reporterArgs"
+  yarn start-server-and-test "node src/platform/testing/e2e/test-server.js --buildtype=vagovprod --port=3001 > /dev/null 2>&1" :3001 "yarn cy:run $reporterArgs --env ci_name=circle"
```

Comment:
I will be exposing the already available `CIRCLECI` environment variable, so you can revert the addition of this flag. #14113

Reply:
👏 nice, will do.